content/vi/docs/reference/glossary/kube-proxy.md

```diff
@@ -0,0 +1,21 @@
+---
+title: kube-proxy
+id: kube-proxy
+date: 2019-29-11
+full_link: /docs/reference/command-line-tools-reference/kube-proxy/
+short_description: >
+  `kube-proxy` là một tầng proxy mạng chạy trên mỗi node trong cluster.
+
+aka:
+tags:
+  - fundamental
+  - networking
+---
+
+[kube-proxy](/docs/reference/command-line-tools-reference/kube-proxy/) là một tầng proxy mạng chạy trên mỗi node trong cluster, triển khai khái niệm về Kubernetes {{< glossary_tooltip term_id="service">}}.
+
+<!--more-->
+
+kube-proxy duy trình network rules trên các node. Những network rules này cho phép việc giao tiếp trên network giữa các pods với trong và ngoài cluster.
```

Comment:
**Must be:**
kube-proxy duy trì các network rules trên các node. Những network rules này cho phép kết nối mạng đến các Pod từ trong hoặc ngoài cluster.